src/vsphere_cpi/spec/integration/nsxt_spec.rb

```diff
@@ -306,6 +377,29 @@ def nsgroup_effective_logical_port_member_ids(nsgroup)
     results.map { |member| member.target_id }
   end
 
+  def create_server_pool(pool_name, nsgroup=nil)
```

Comment:
I'd rather we have two methods to create the two kinds of server pools. Also spaces around `=`.